### ISSUE TITLE
Notebook on polyaxon should enables us download files.

### ISSUE THREAD

**Issue by yu-iskw** [polyaxon/polyaxon#507]
### Feature motivation
One of the challenge to use notebook is to synchronize notebooks between on polyaxon and in local. As well as, I want to download trained models from notebook on polyaxon. However, when I tried to download file from remote notebooks, I was faced with 403 forbidden.

![failed-download](https://user-images.githubusercontent.com/1523515/61250282-2abd3580-a70c-11e9-8578-49e2a7e79b41.png)

### Feature description
It would be great to download files from notebooks.

### Feature implementation
I seems that we can improve it by adding a option to `jupyter notebook`.

**Comment by mouradmourafiq** [comment]
Notebook experience is one of the things that need to be improved for sure, there are many things steps that I am already thinking about.

Was this a file that you tried to download or a model? is it possible to attach a very simple script and some steps to reproduce the issue. 

Thanks for reporting.

**Comment by yu-iskw** [comment]
I usually use a script to upload files to GCS using polyastore. But it is a little bothersome to me. 

Technically, we may be able to resolve the issue like below. 
```
# Or, jupyter notebook option
jupyter notebook ... --NotebookApp.allow_origin='*'
```

**Comment by mouradmourafiq** [comment]
It should be part of the v0.5.4 release, thanks for the flag suggestion.

**Comment by yu-iskw** [comment]
Thanks for fixing it immediately!

**Comment by mouradmourafiq** [comment]
Reopening this, it's seems to only solve the issue when using JupyterLab and not notebooks.

**Comment by mouradmourafiq** [comment]
Closed as well in #690 , Polyaxon has a much better implementation for proxing request wihtout limiting requests.
